sql/patch/00_STAT_STATEMENTS_DATA.sql

```diff
@@ -0,0 +1,8 @@
+SET statement_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+ALTER TABLE stat_statements_data
+   ADD COLUMN ssd_user_id integer DEFAULT 0;
```

Comment:
"ADD COLUMN + DEFAULT 0" in one statement wouldn't work also in regards to existing DBs. In our company we have for example gigabytes of old  statement data and it makes no big sense to overwrite that. Doing "SET DEFAULT" in a separate command would alleviate the problem.